app/helpers/navigation_helper.rb

```diff
@@ -1,4 +1,22 @@
 module NavigationHelper
+  NAVIGATION_LINKS = {
+    agenda: "/",
+    schedule: "/",
+    profile: "/profile",
+    notifications: "/",
+    about: "/"
+  }
+
+  def nav_icon_class_for(item)
+    return "fill-red w-6 h-6" if current_page?(NAVIGATION_LINKS[item])
```

Comment:
I think using [`class_names`](https://til.hashrocket.com/posts/yitpbvf6dw-easy-conditional-style-class-names-in-rails) is a better approach

Reply:
I updated the approach, I think this works best. Let me know what you think 👍